EIPS/eip-5792.md

```diff
@@ -12,61 +12,23 @@ created: 2022-10-17
 
 ## Abstract
 
-Defines new JSON-RPC methods which enable apps to ask a wallet to process a batch of onchain write calls and to check on the status of those calls. Calls can be specified to make use of various wallet-defined capabilities, and there is also a new RPC to enable apps to discover which capabilities a wallet supports. 
+Defines new JSON-RPC methods which enable apps to ask a wallet to process a batch of onchain write calls and to check on the status of those calls. Dapps can specify that these onchain calls be executed taking advantage of specific capabilities previously expressed by the wallet; an additional, novel wallet RPC is defined to enable apps to query the wallet for those capabilities.
```

Comment:
Calls are not necessarily made by a "dapp"

Reply:
Tbh I think the whole thing here reads less clearly :)

Reply:
You're right about dapps-- let's fix that whatever replaces this line.

Sorry for the torturous syntax-- it's been my experience, though, that people allocate a number of minutes they think a doc will take to read and start skimming near the end, so without an abstract that warns them there's a whole section on capability expression near the end, they might not allocate the time/mental bandwidth to get that far before thinking they got the gist.  i'm not wed to the specific language, but AM wed to the abstract signaling that whole section.

How's this, at the other end of the verbosity spectrum from the previous proposal (any co-author drops an emoji thumb and second thumb-up can just commit the suggestion?):

```suggestion
Defines a new JSON-RPC method for apps to send batches of onchain write calls together, which returns a pointer to the execution state, as well as a second method to query that state.  A third application-to-wallet method is included for requesting the capability to use the first two.
```